pyvista/utilities/fileio.py

```diff
@@ -187,7 +199,13 @@ def read(filename, attrs=None):
             reader = get_reader(filename)
             return standard_reader_routine(reader, filename)
         except KeyError:
-            pass
+            # Attempt read with meshio
+            try:
+                from meshio._helpers import _extension_to_filetype
```

Comment:
Can we put this in the `read_meshio` method and default the `file_format` argument to `None`, then if it is `None`, look it up here?

Reply:
Moving this would resolve my comment below about the warning message if meshio is not installed

Reply:
Might be useless. I'll remove it since this dict is used to guess type in ``meshio``.